apis/refs/v1beta1/helper.go

```diff
@@ -30,3 +30,14 @@ func GetResourceID(u *unstructured.Unstructured) (string, error) {
 	}
 	return resourceID, nil
 }
+
+func GetLocation(u *unstructured.Unstructured) (string, error) {
```

Comment:
We don't want to make this a helper function, see: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2337#discussion_r1696037535

Reply:
But it's going to be used a fair amount I think for any ref that has `locations` in its fully qualified name